src/m365/spo/commands/applicationcustomizer/applicationcustomizer-add.ts

```diff
@@ -110,6 +115,10 @@ class SpoApplicationCustomizerAddCommand extends SpoCommand {
       ClientSideComponentId: args.options.clientSideComponentId
     };
 
+    if (args.options.description) {
+      requestBody.Description = args.options.description;
+    }
```

Comment:
Instead of creating an extra condition, let's merge it directly with the request body like the other options.